scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/TaskLauncherRebinder.java

```diff
@@ -31,6 +31,7 @@
 import org.objectweb.proactive.api.PAActiveObject;
 import org.objectweb.proactive.core.node.Node;
 import org.objectweb.proactive.core.node.NodeFactory;
+import org.ow2.proactive.resourcemanager.core.properties.PAResourceManagerProperties;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.ow2.proactive.resourcemanager.core.properties.PAResourceManagerProperties'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)